cmds/get_kube_ca.go

```diff
@@ -25,7 +25,7 @@ func NewCmdGetKubeCA(clientGetter genericclioptions.RESTClientGetter) *cobra.Com
 			if err != nil {
 				Fatal(errors.Wrap(err, "failed to load tls files"))
 			}
-			if cfg.Insecure {
+			if cfg.Insecure || len(cfg.CAData) == 0 {
```

Comment:
zero-length CAData doesn't mean the connection is insecure, it means the system roots are used

Reply:
Thanks @liggitt ! I am happily surprised to see you here. I have reverted this already https://github.com/kubepack/onessl/commit/307652be45049eb6ee601ad091289a56880c03dd .